doc/cross-compile/buildroot.md

```diff
@@ -0,0 +1,39 @@
+# How to make cross-compilation work with buildroot
+
+1. install the build dependencies on your host PC, like: cmake, g++, clang...
+2. install the `rust` toolchain:
+
+   ```console
+   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+   ```
+
+3. add the arm64 target for Rust: `rustup target add aarch64-unknown-linux-gnu`
+4. modify the arm64 target name to suit your cross-compilation tool, need to
+   create this file `~/.cargo/config.toml`, add:
+   ```console
+   [target.aarch64-unknown-linux-gnu]
+   linker = "aarch64-buildroot-linux-gnu-gcc"
+   ```
+   "aarch64-buildroot-linux-GNU-gcc" is your real cross-compilation tool name
```

Comment:
I think the markdown linter complains about not having empty lines before and after the code block. You can run the markdown linter by using this command
```console
internal/scripts/check_markdown_linting.sh
```

Please also use an inline code block for `"arch64..."`